CHANGELOG.md

```diff
@@ -97,6 +97,12 @@ Backwards compatible changes
   swap-involutive : swap ∘ swap ≗ id
   ```
 
+* Added new proofs to `Data.List.Properties`:
+  ```agda
+  tabulate-cong : ∀ {n a} {A : Set a} {f g : Fin n → A} → f ≗ g → tabulate f ≡ tabulate g
+  tabulate-lookup : ∀ {a} {A : Set a} {xs : List A} → tabulate (lookup xs) ≡ xs
+  ```
```

Comment:
We usually drop the implicit arguments in these types to make it
more readable (cf. rest of the CHANGELOG)

Reply:
I hadn't noticed, fixed.